modules/oauth-customizing-the-oauth-server-URL.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * authentication/configuring-internal-oauth.adoc
+
+[id="customizing-the-oauth-server-url_{context}"]
+= Customizing oAuth server URL
+
+Custom route functionality for the oAuth server uses the `ingress` config route configuration API. A custom hostname and a TLS certificate can be set using the `spec.componentRoutes` part of the configuration.
+
+[id="customizing-the-oauth-server-route_{context}"]
+== Customizing the oAuth server route
+
+.Prerequisites
+
+* Log in to the cluster as a user with administrative privileges.
```

Comment:
Technically you just need to be able to update the spec of the `ingress.config.openshift.io/cluster` resource, but this is probably fine for documentation purposes.

Reply:
Ok, so leaving it as is.